frontend/src/app/home/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Link from "next/link";
+import Header from "@/components/ui/Header";
+
+const Navbar = () => {
+  return (
+    <nav className="flex justify-between items-center p-4 bg-gray-900">
+      <div className="flex-shrink-0">
+        <Header />
+      </div>
+
+      <div className="flex space-x-6">
```

Comment:
Consideration: Add highlight to indicate which link we are currently on (as designed on figma)

![image](https://github.com/user-attachments/assets/b47bff00-8a0b-466d-a01a-fea2acc1b8d8)